index.bs

```diff
@@ -2623,6 +2631,69 @@ To <dfn>get the navigation info</dfn>, given |context| and |navigation status|:
 
 ### Commands ### {#module-browsingContext-commands}
 
+#### The browsingContext.captureScreenshot Command ####  {#command-browsingContext-captureScreenshot}
+
+The <dfn export for=commands>browsingContext.captureScreenshot</dfn> command
+captures an image of the given browsing context, and returns it as a
+Base64-encoded string.
+
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      BrowsingContextCaptureScreenshotCommand = {
+        method: "browsingContext.captureScreenshot",
+        params: BrowsingContextCaptureScreenshotParameters
+      }
+
+      BrowsingContextCaptureScreenshotParameters = {
+        context: BrowsingContext
+      }
+      </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl local-cddl">
+        BrowsingContextCaptureScreenshotResult = {
+          data: text
+        }
+    </pre>
+   </dd>
+</dl>
+
+TODO: Full page screenshots, and multiple output formats
```

Comment:
I assume we also want to allow to capture screenshots for specific elements only?

Reply:
Right, I was imagining that as part of a different module since it would be about elements. But we could of course divide things up in more than one way e.g. we could have a screenshots module with a command that would either take a context or an element.

In any case that TODO was only supposed to be for things that are unambiguously in the remit of the browsing context screenshot feature.

Reply:
Would you mind filing a new issue for that and reference it here?